
# DNA-GCN: Graph convolutional networks for predicting DNA-protein binding

[arXiv](https://arxiv.org/abs/2106.01836), [PDF](https://arxiv.org/pdf/2106.01836.pdf)

## Authors

- Yuhang Guo
- Xiao Luo
- Liang Chen
- Minghua Deng

## Abstract

Predicting DNA-protein binding is an important and classic problem in bioinformatics. Convolutional neural networks have outperformed conventional methods in modeling the sequence specificity of DNA-protein binding. However, none of the studies has utilized graph convolutional networks for motif inference. In this work, we propose to use graph convolutional networks for motif inference. We build a sequence k-mer graph for the whole dataset based on k-mer co-occurrence and k-mer sequence relationship and then learn DNA Graph Convolutional Network (DNA-GCN) for the whole dataset. Our DNA-GCN is initialized with a one-hot representation for all nodes, and it then jointly learns the embeddings for both k-mers and sequences, as supervised by the known labels of sequences. We evaluate our model on 50 datasets from ENCODE. DNA-GCN shows its competitive performance compared with the baseline model. Besides, we analyze our model and design several different architectures to help fit different datasets.

## Comments

10 pages, 3 figures

## Source Code

Official Code

- [https://github.com/Tinard/dnagcn](https://github.com/Tinard/dnagcn)

Community Code

- [https://paperswithcode.com/paper/dna-gcn-graph-convolutional-networks-for](https://paperswithcode.com/paper/dna-gcn-graph-convolutional-networks-for)

## Bibtex

```tex
@misc{guo2021dnagcn,
      title={DNA-GCN: Graph convolutional networks for predicting DNA-protein binding}, 
      author={Yuhang Guo and Xiao Luo and Liang Chen and Minghua Deng},
      year={2021},
      eprint={2106.01836},
      archivePrefix={arXiv},
      primaryClass={q-bio.GN}
}
```

## Notes

Type your reading notes here...

